README.md

```diff
@@ -213,149 +187,47 @@ For Example:
             /openldap-modify-users.yaml <- openldap overlay
  ```
 
-### Running
+## Creating and Managing deployment
 
-All configs needed by ansible are also needed to be mounted into the docker container. In general any file/folder path set via an ansible flag are equivalent to the file/folder being mounted to the docker container at /config/<lower_case_variable_name>.
+Create and manage deployments by either: 
 
-Examples:
+- running [Ansible](docs/user/AnsibleUsage.md) directly on your workstation, or
```

Comment:
Reverse the order so our preferred method is first in the list and add recommended

- running [Docker container](docs/user/DockerUsage.md) - Recommended, or
- running [Ansible](docs/user/AnsibleUsage.md) directly on your workstation

Reply:
updated